vspec.py

```diff
@@ -240,17 +241,22 @@ def convert_yaml_to_list(raw_yaml):
 
     return lst
 
-
-
-def load_tree(file_name, include_paths, exclude_private=False, break_on_noncore_attribute=False):
+# As of today the VSS language/metamodel specification contain some attributes (e.g. arraysize) 
+# that is ignored by most tools in this repository. A tool can specify which VSS core attributes that it
```

Comment:
The point about it being ignored by most tools today seems not relevant, and I would like to remove it.

But idea and documentation that tools can specify the unsupported attributes seems reasonable **if** some attributes might not serve a purpose for certain tools?  But do we have such an today, a tool that actually does not need a certain piece of metadata?  

The original reason stated (is ignored by some/most tools) seems for most cases to be a bug in the tool, rather than a logical behavior?

In summary I support introducing this feature only if we know a case for it today.  Otherwise it is better to wait until that case comes up.

Reply:
Can we assume that all tools shall support all metamodel elements? If we consider it as natural that csv/json (but possibly not protobuf as it is a "contrib" tool) shall support "arraysize", then I agree that we instead shall create issues in vss-tools on the bug that csv/json tooling does not support arraysize.